src/wasi.ts

```diff
@@ -37,641 +31,653 @@ export default class WASI {
     this.args = args;
     this.env = env;
     this.fds = fds;
-    let self = this;
-    this.wasiImport = {
-      args_sizes_get(argc: number, argv_buf_size: number): number {
-        let buffer = new DataView(self.inst.exports.memory.buffer);
-        buffer.setUint32(argc, self.args.length, true);
-        let buf_size = 0;
-        for (let arg of self.args) {
-          buf_size += arg.length + 1;
-        }
-        buffer.setUint32(argv_buf_size, buf_size, true);
-        //console.log(buffer.getUint32(argc, true), buffer.getUint32(argv_buf_size, true));
-        return 0;
-      },
-      args_get(argv: number, argv_buf: number): number {
-        let buffer = new DataView(self.inst.exports.memory.buffer);
-        let buffer8 = new Uint8Array(self.inst.exports.memory.buffer);
-        let orig_argv_buf = argv_buf;
```

Comment:
This one is used by the commented out console.log below. Maybe keep it as comment?

Reply:
Would you be open to a PR that converts all the console.log statements to a configurable debug logger? That way logging can be turned off and on without modifying the source.

Reply:
Yes

Reply:
I converted the console.log to a simple debug logger. I kept it in this PR since it's also geared towards the contributor onboarding experience.